### PR TITLE
Update Korean doc

### PR DESCRIPTION
**Pull request by jeyong**
Update Korean doc and merge it.

### CODE REVIEW

File: kr/advanced/rtk_gps.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# RTK GPS (Background)
```
Comment (hamishwillee) [inline kr/advanced/rtk_gps.md:1]: FYI Not translated.

File: kr/middleware/micrortps.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+   # Add new topic...
+   )
+
+set(config_rtps_receive_topics
+   sensor_baro
+   # Add new topic...
+   )
+```
+
+> **Caution** At time of writing (August 2017), only the small set of uORB topics listed above are included in our cmake files: **posix_sitl_default.cmake**, **nuttx_px4fmu-v4_default.cmake**, **posix_sdflight_default.cmake**. It is likely you will need to edit your *cmake* file and add additional uORB topics. In future we hope to define a larger standard set.
```
Comment (hamishwillee) [inline kr/middleware/micrortps.md:62]: Only translated up to here.

File: kr/middleware/micrortps_throughput_test.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# Micro RTPS Throughput Test
```
Comment (hamishwillee) [inline kr/middleware/micrortps_throughput_test.md:1]: Not translated

File: kr/middleware/modules_estimator.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Modules Reference: Estimator
+# Modules 레퍼런스: Estimator
```
Comment (hamishwillee) [inline kr/middleware/modules_estimator.md:1]: FYI only, all this module text is auto-generated from code. That shouldn't change your process.

File: kr/ros/external_position_estimation.md [hunk 5]
```diff
@@ -31,3 +31,61 @@ mavros ROS-Mavlink 인터페이스는 기본적으로 이런 메시지를 보내
...
+
+* robot을 손으로 움직이고 estimated local position이 일정한지를 봅니다.(항상 NED로)
+
+* 수직 축으로 robot을 회전시키고 Mavlink inspector로 yaw를 검사합니다.
+
+이런 단계들을 거치면, 처음 비행을 시도할 수 있습니다.
+
+robot을 그라운드에 위치시키고 mocap 피드백을 스트림으로 내보내도록 합니다. 왼쪽 스틱을 낮추고 모터에 시동을 겁니다.
+
+At this point, with the left stick at the lowest position, switch to position control. You should have a green light. The green light tells you that position feedback is available and position control is now activated.
```
Comment (hamishwillee) [inline kr/ros/external_position_estimation.md:85]: Bit below here not translated

File: kr/setup/_gcc_toolchain_installation.txt [hunk 6]
```diff
@@ -0,0 +1,54 @@
+<!-- GCC toolchain documentation used for all Linux platforms to build NuttX -->
```
Comment (hamishwillee) [inline kr/setup/_gcc_toolchain_installation.txt:1]: Not translated

File: kr/setup/_ninja_build_system.txt [hunk 7]
```diff
@@ -0,0 +1,14 @@
+## Ninja Build System
```
Comment (hamishwillee) [inline kr/setup/_ninja_build_system.txt:1]: Not translated

File: kr/setup/dev_env_advanced_linux.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+# Advanced Linux
```
Comment (hamishwillee) [inline kr/setup/dev_env_advanced_linux.md:1]: Not translated.

File: kr/setup/dev_env_linux_ubuntu.md [hunk 9]
```diff
@@ -0,0 +1,394 @@
...
+
+다음은 지원하는 타겟의 각 개발환경을 *수동*으로 셋업하는 방법을 설명합니다.
+
+> **Tip** We recommend that you use the [Convenience bash scripts](#convenience-bash-scripts) to install the Simulators and/or NuttX toolchain (this is easier than typing in the instructions below). Then follow just the additional instructions for other targets (e.g. Qualcomm Snapdragon Flight, Bebop, Raspberry Pi, etc.)
+
+
+
+## 편리한 Bash 스크립트
+
+We've created a number of bash scripts that you can use to install the Simulators and/or NuttX toolchain. All the scripts include the *Qt Creator IDE*, [Ninja Build System](#ninja-build-system), [Common Dependencies](#common-dependencies), [FastRTPS](#fastrtps-installation), and also download the PX4 source to your computer (**~/src/Firmware**).
```
Comment (hamishwillee) [inline kr/setup/dev_env_linux_ubuntu.md:13]: Not translated below here.

File: kr/setup/dev_env_windows.md [hunk 10]
```diff
@@ -2,27 +2,73 @@
...
-  * [PX4 Toolchain Installer v14 윈도우용 다운로드](http://firmware.diydrones.com/Tools/PX4-tools/px4_toolchain_installer_v14_win.exe) (32/64 bit 시스템, 완전한 빌드 시스템, 드라이버)
-  * [PX4 USB Drivers](http://pixhawk.org/static/px4driver.msi) (32/64 bit 시스템)
+* The native toolchain allows you to build for NuttX/Pixhawk and jMAVSim simulator targets.
+* The Windows Bash toolchain allows you to build for NuttX/Pixhawk targets (only).
 
-이제 [처음 빌드하기](../setup/building_px4.md)를 이어서 실행하면 됩니다!
+### Native 툴체인
 
-## 새소식! 윈도우용 Bash
+Download and install these on your system:
```
Comment (hamishwillee) [inline kr/setup/dev_env_windows.md:14]: Partially translated

File: kr/sik-radio.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
+# SiK Radio
```
Comment (hamishwillee) [inline kr/sik-radio.md:1]: Not translated.

File: kr/simulation/README.md [hunk 12]
```diff
@@ -0,0 +1,227 @@
+# Simulation
```
Comment (hamishwillee) [inline kr/simulation/README.md:1]: Not translated. This page is actually pretty cool/important.

File: kr/simulation/jmavsim.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
+# jMAVSim with SITL
```
Comment (hamishwillee) [inline kr/simulation/jmavsim.md:1]: Not translated. This page was previously named sitl.md, but has been updated "a little"

File: kr/tutorials/linux_sbus.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
+# S.Bus Driver for Linux
```
Comment (hamishwillee) [inline kr/tutorials/linux_sbus.md:1]: No translated

**Comment by hamishwillee** [comment]
@jeyong WOW, you have done a lot of work :-).
There are some pages that are not translated. Did you want me to wait on those, or merge this stuff here first?

**Comment by jeyong** [comment]
Yep I know it. But many pages in Korean docs are deprecated. So if you
don't mind, merge it first. Then I will catch up.

**Comment by hamishwillee** [comment]
Of course not - that's fine, merged. 

Just FYI, there are a couple of broken links:
link to "simulation/sitl.md" on page "test_and_ci/integration_testing.md" could not be resolved
link to "simulation/jmavsim.md" on page "simulation/README.md" could not be resolved
link to "simulation/sitl.md" on page "test_and_ci/integration_testing.md" could not be resolved

The sitl.md links usually move to jmavsim.md.

**Comment by hamishwillee** [comment]
And thanks again, this is really excellent.

**Pull request merged**